### DISCUSSION TITLE
Thumbnail previews in Windows explorer show "PDF" over non-pdf files associated with Sumatra

### DISCUSSION THREAD

**Discussion by UntypedLetter** [sumatrapdfreader/sumatrapdf#4583] [Q&A]
After installing DarkThumbs and PowerToys, I can successfully view thumbnails of many ebook formats in Windows explorer; however, all file formats that are associated with Sumatra show a "PDF" overlayed or superimposed over the thumbnail previews even if they are not pdf files. Shouldn't it show a different indicator or icon for each file format? Is this a bug?

<img width="672" alt="Sumatra thumbnail issue" src="https://github.com/user-attachments/assets/865d3cdd-70b4-4f42-8095-8c8666aca87a">

This happens if I select any of the following in Windows explorer, on the "View" tab:

- Extra large icons
- Large icons
- Medium icons

But if I select any of the following in Windows explorer, on the "View" tab, the icons change accordingly for each file type:

- Small icons
- List Details

**Comment by GitHubRulesOK** [comment]
some issues can be down to delayed Windows icon cache cleaning , but unknown if that's your case,
Windows will often use what it thinks is appropriate based on association so I have a mix of MuPDF and SumatraPDF
![image](https://github.com/user-attachments/assets/9bd7c4fd-85f9-47dc-9ecf-907c8eec817a) 
![image](https://github.com/user-attachments/assets/9f7c5788-d362-43d1-8961-852524396ece)
 details / small or Large it will remember the association at that time
![image](https://github.com/user-attachments/assets/d5bdd345-0772-4a3a-8998-e1d22c84056e)

However I do see what you describe in that these are small but different as medium icons
![image](https://github.com/user-attachments/assets/dc92a398-c18a-42cb-8c0c-bc17c21756f7) 
![image](https://github.com/user-attachments/assets/5602e67f-7a01-4cc9-9b5e-1da120439353)

**Reply by UntypedLetter** [reply]
Windows had a mind of its own, long before AI. xD
Yes, you have understood it correctly. I feel this should be treated as an issue.
